### DISCUSSION TITLE
Can the scorecard compare only 2 models?

### DISCUSSION THREAD

**Discussion by bonnystrong** [NOAA-GSL/MATS#785] [Scorecard - user discussions]
From Steve W on Aug 26, 2022:
Is the scorecard envisioned to just compare two different model systems (retro or R/T? I (Steve) am assuming so (too many DOFs to do anything else). – [thanks for discussion on this].

**Comment by bonnystrong** [comment]
From Randy: at the moment only two. a “data source” and a “validation data source”

With further discussion, the idea of a scorecard is to compare only 2 models

**Comment by randytpierce** [comment]
Yes, that is certainly correct. It might be possible to do three but I cannot quite visualize how we would work out the symbols, three colors perhaps?
